Penteract/Assets/Scripts/AIMeleeGrunt.cpp

```diff
@@ -225,10 +246,20 @@ void AIMeleeGrunt::OnCollision(GameObject& collidedWith, float3 collisionNormal,
 			bool hitTaken = false;
 			if (collidedWith.name == "FangBullet") {
 				hitTaken = true;
-				gruntCharacter.GetHit(playerController->playerFang.damageHit + playerController->GetOverPowerMode());
+				if(state == AIState::STUNNED && EMPUpgraded){
+					gruntCharacter.GetHit(99);
```

Comment:
It would be nice to have the 99 dmg transformed into a const or def so is no longer a magic number

Reply:
Noted